yellowbrick/classifier/base.py

```diff
@@ -157,18 +201,131 @@ def score(self, X, y, **kwargs):
         ----------
         X : array-like
             X (also X_test) are the dependent variables of test set to predict
+
         y : array-like
             y (also y_test) is the independent actual variables to score against
 
         Returns
         -------
         score : float
+            Returns the score of the underlying model, usually accuracy for
+            classification models. Refer to the specific model for more details.
         """
-        self.score_ = self.estimator.score(X, y, **kwargs)
+        # If the estimator has been passed in fitted but the visualizer was not fit
+        # then we can retrieve the classes from the estimator, unfortunately we cannot
+        # retrieve the class counts so we simply set them to None and warn the user.
+        # NOTE: cannot test if hasattr(self, "classes_") because it will be proxied.
+        if not hasattr(self, "class_counts_"):
```

Comment:
This allows for fitted models to be instantiated and the `fit()` call bypassed; though the `check_fitted` functionality means that we should ask users to always call fit.